### PR TITLE
Update gradle dependencies

### LINKED ISSUE #1493: Obsolete Gradle Dependencies
AndroidLint: Correctness warning
Few issues
1.Obsolete Gradle dependencies
2.Restricted API should be suppressed(@SuppresLint) as there seems no workaround for it.

Would you like to work on this issue?
Yes, I want to work on this issue.

### PR DESCRIPTION
**Pull request by gupta1anubhav**
## Description (required)

Gradle Dependencies are outdated as new versions are available.

Fixes #1493 

I have updated the Gradle dependencies to the latest version.
Screenshots before change

screenshot from 2018-05-23 15-07-42

## Tests performed (required)

Tested on Redmi Note 4 with and working on all build versions.

## Screenshots showing what changed (optional)
 
Screenshots after change
screenshot from 2018-05-23 15-15-48

### CODE REVIEW

**Comment by gupta1anubhav** [comment]
Sir @maskaravivek , I have made the appropriate changes to the description of PR.

File: app/build.gradle [hunk 1]
```diff
@@ -20,19 +20,19 @@ dependencies {
...
         transitive=true
     }
 
     implementation "com.github.deano2390:MaterialShowcaseView:1.2.0"
 
-    implementation "com.android.support:support-v4:$SUPPORT_LIB_VERSION"
-    implementation "com.android.support:appcompat-v7:$SUPPORT_LIB_VERSION"
-    implementation "com.android.support:design:$SUPPORT_LIB_VERSION"
-    implementation "com.android.support:customtabs:$SUPPORT_LIB_VERSION"
+    implementation "com.android.support:support-v4:27.1.1"
```
Comment (misaochan) [inline app/build.gradle]: You probably want to set `$SUPPORT_LIB_VERSION` to 27.1.1 instead of hardcoding all of it. ;)

**Comment by gupta1anubhav** [comment]
@misaochan I have made updated set $SUPPORT_LIB_VERSION to 27.1.1 .
And ProdRelease is working fine for the main functions of the app after changing gradle.

**Comment by neslihanturan** [comment]
And please no more FixForXXX title. It is easier to test if you write as: Fix for #xxx TITLE_OF_XXX

**Comment by gupta1anubhav** [comment]
ok

**Pull request merged**